docs/upgrading_to_v1.0.md

```diff
@@ -0,0 +1,35 @@
+# Upgrading to v1.0
+The v1.0 release of *cloud-spanner* is a backwards incompatible
+release.
+
+### Google Cloud Platform Provider upgrade
+The Terraform Cloud Spanner Module now requires version 6.1 or higher of the Google Cloud Platform Providers.
+
+```diff
```

Comment:
nit: We can probably leave off this sample block, as most probably don't use their own, and the module will enforce it anyway.  Do leave the `Google Cloud Platform Provider upgrade` section.